api/integrations/codebase/zulip_codebase_mirror

```diff
@@ -77,7 +77,7 @@ def make_api_call(path):
     # type: (str) -> Optional[List[Dict[str, Any]]]
     response = requests.get("https://api3.codebasehq.com/%s" % (path,),
                             auth=(config.CODEBASE_API_USERNAME, config.CODEBASE_API_KEY),
-                            params={'raw': True},
+                            params={'raw': 'True'},
```

Comment:
I suspect that the casting to string feature of python-requests is intended, but worth opening a PR.  I think we can change this to `'True'` in Zulip regardless; it's not wrong to do so.  But I do think it's worth having the conversation with typeshed upstream since requests is a major library, so wrong stubs here will impact a lot of other people too.